sdk/resources/Microsoft.Azure.Management.Resource/src/generate.ps1

```diff
@@ -3,10 +3,10 @@
 # and copy these new set of files to the main Generated folder. This way exisiting files will not be deleted.
 
 # Generate package with resources tag
-#Start-AutoRestCodeGeneration -ResourceProvider "resources/resource-manager" -SdkRepoRootPath "$PSScriptRoot\..\..\..\.." -AutoRestVersion "v2" -AutoRestCodeGenerationFlags "--tag=package-resources-2020-10" -SdkGenerationDirectory "$PSScriptRoot\Generated\Resources"
+Start-AutoRestCodeGeneration -ResourceProvider "resources/resource-manager" -SdkRepoRootPath "$PSScriptRoot\..\..\..\.." -AutoRestVersion "v2" -AutoRestCodeGenerationFlags "--tag=package-resources-2020-10" -SdkGenerationDirectory "$PSScriptRoot\Generated\Resources"
```

Comment:
You should not need to specify the tag if you are using the default

Reply:
That's good to know! However, I don't think we can do that for the Resources RP because there's no unique default in the readme (see: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/readme.md#validation).

Reply:
Yeah, your team needs to fix this in the long term - there should be a default tag, which corresponds to the latest or latest stable api.  This is ok for now